apps/users/templates/advocate_europe_users/base_accounts_settings_form.html

```diff
@@ -4,20 +4,27 @@
 {% block content %}
     {% force_translation 'en' %}
     <div class="container">
-        <div class="row profile-form">
+        <div class="row account-settings">
             <div class="col-md-3 well">
                 <h1>{% trans 'Account settings' %}</h1>
                 <ul class="list-group">
                     {% url 'edit_profile' as edit_profile_url %}
                     <li class="list-group-item {% if edit_profile_url == request.get_full_path  %} active {% endif %}">
                         <a href="{% url 'edit_profile' %}">
-                            <i class="fa fa-user" aria-hidden="true"></i> {% trans 'Your Profile' %}
+                            <span class="fa-stack">
```

Comment:
While the stacking aligns fas and links perfectly, I am not quite sure if it's a good idea to use transparent icons just to achieve this.